lib/render/pxrUsdMayaGL/batchRenderer.cpp

```diff
@@ -130,6 +140,13 @@ UsdMayaGLBatchRenderer::GetDelegatePrefix(const bool isViewport2) const
 bool
 UsdMayaGLBatchRenderer::AddShapeAdapter(PxrMayaHdShapeAdapter* shapeAdapter)
 {
+    TRACE_FUNCTION();
+
+    MProfilingScope profilingScope(
+        ProfilerCategory,
+        MProfiler::kColorG_L3,
+        "Batch Renderer Adding Shape Adapter");
```

Comment:
Since you are in the batch renderer category, what's the advantage of adding `Batch Renderer` to every event name? It makes names very long.

Reply:
I was finding the full event name useful when looking at the CPU or thread views in the Profiler vs. the category view. That way I could see batch renderer performance in the context of a full scene render and not have to turn off any other categories.

Reply:
Some additional tricks related to profiler and category view. Explore the option to collapse events. This reduces the noise by collapsing all events to starting one on a thread. What will remain is a view of the current concurrency level.

You can find this option in the right-click menu, acting on selected events.